### DISCUSSION TITLE
pag播放问题

### DISCUSSION THREAD

**Discussion by xiangpidan** [Tencent/libpag#2065] [Q&A] [answered]
从AE中输出PAG后再本机是可以播放的，给到开发后开发播放不了不知道什么原因

**Comment by kevingpqi123** [comment, marked as answer]
建议让你们的开发同学基于 pag 的官方 demo 做测试，如果 pag 的官方 demo 播放是正常的，排查自己的使用问题；如果官方 demo 播放异常，将 pag 文件附在问题中
https://pag.art/docs/pad-demo-download.html
